using_images/s2i_images/java.adoc

```diff
@@ -85,14 +86,19 @@ You can override these default goals and options by specifying the following env
 
 |Variable name |Description
```

Comment:
Hi @hmaclean73 , sure sorry, I was off last week and I'm catching up. I'm slating this for my morning Tomorrow.

Reply:
Hi @hmaclean73 , I've raised a PR in your fork for the branch in this PR (if that makes sense) - https://github.com/hmaclean73/openshift-docs/pull/2 - which adds a table of all the environment variables. (actually, two tables)

Because the tables are so huge, rather than overwhelm the section in the middle of the document where some env vars were listed (which were pertinent to the example prior), I've added them as a new section at the end, and referenced that section from the earlier example. Feel free to choose a totally different structure!

There's definitely some room for improvement in how we present the variables. In particular the ones that begin "prefix_" are somewhat complex and could benefit from being broken out and explained separately perhaps. But I'm not sure whether that should block the other work.